mern/server/utils/parse.js

```diff
@@ -0,0 +1,72 @@
+import QA from "../models/question_answer.js";
```

Comment:
Looks good! I like the use of helper functions to make the original methods shorter (reduces bloating) 👍 
In terms of clean architecture, we should consider if this is a good class (during call with Ekagra, he mentioned thinking of one js file as a class). In my opinion I think this is fine since it follows the Single Responsibility Principle. We would only use these functions for one cause, parsing the transcripts.

Reply:
LGTM